libs/live-nft-react/src/hooks/useCheckNftAccount.ts

```diff
@@ -9,19 +9,32 @@ import { nftsByCollections } from "@ledgerhq/live-nft/index";
 import { hashProtoNFT } from "./helpers";
 
 /**
- * useCheckNftAccount() will apply a spam filtering on top of existing NFT data.
- * - addresses: a list of wallet addresses separated by a ","
- * - nftOwned: the array of all nfts as found by all user's account on Ledger Live
- * - chains: a list of selected network to search for NFTs
- * - action: custom action to handle collections
- * NB: for performance, make sure that addresses, nftOwned and chains are memoized
+ * A React hook that checks NFT accounts against specified criteria and provides filtering functionality for managing NFT collections.
+ *
+ * @param {Object} params - The parameters for the hook.
+ * @param {string} params.addresses - A comma-separated string of NFT addresses to check.
+ * @param {Array} params.nftsOwned - An array of owned NFTs.
+ * @param {Array} params.chains - An array representing the blockchain chains.
+ * @param {number} params.threshold - A numeric threshold for filtering NFTs.
+ * @param {Function} params.action - A callback function to execute when spam is detected.
+ * @param {boolean} [params.enabled=false] - A flag to enable or disable the hook's functionality.
+ *
+ * @returns {Object} The result of the hook.
+ * @returns {Array} returns.nfts - An array of filtered NFTs.
+ * @returns {Object} returns.queryResult - The result of the infinite query, containing pagination and loading states.
+ *
  */
+
+export const ONE_DAY = 24 * 60 * 60 * 1000;
+export const HALF_DAY = ONE_DAY / 2;
```

Comment:
perhaps we could consider a file with all the timers, as this one is specific to the market, for example ? wdyt?

Reply:
Yes it's good too but what I suggested here here was either:

- To import `ONE_DAY` and `HALF_DAY` from `ledger-live-common/src/market/utils/timers`

- Or to remove `HALF_DAY` from `ledger-live-common/src/market/utils/timers` and remove the `export` from the declaration:
   ```suggestion
   const ONE_DAY = 24 * 60 * 60 * 1000;
   const HALF_DAY = ONE_DAY / 2;
   ```


It's a detail but since `HALF_DAY` is not imported anywhere it looks like extra unused code right now. Unless of course if you're planning to use it later.